data-protocols/ids/ids-api-multipart-dispatcher-v1/src/test/java/org/eclipse/dataspaceconnector/ids/api/multipart/client/MultipartDispatcherIntegrationTest.java

```diff
@@ -145,7 +145,11 @@ void testSendArtifactRequestMessage(RemoteMessageDispatcherRegistry dispatcher)
 
     @Test
     void testSendContractOfferMessage(RemoteMessageDispatcherRegistry dispatcher) {
-        var contractOffer = ContractOffer.Builder.newInstance().id("id").policy(Policy.Builder.newInstance().build()).build();
+        var contractOffer = ContractOffer.Builder.newInstance()
```

Comment:
minor nit: building the offer could be moved to a separate method, to avoid code duplication. We often create two methods, one returning the object itself, when just a generic default is needed, and another one, returning the `Builder`, so that we can easily customize the object.